core/src/main/java/com/graphhopper/routing/util/FlagEncoder.java

```diff
@@ -53,6 +54,14 @@
      */
     DecimalEncodedValue getAverageSpeedEnc();
 
+    /**
+     * Decide whether a restriction relation is enabled for a given mode of travel. This skips some relations before
+     * handleRelationTags is called.
+     *
+     * @return true if this encoder allows travel.
+     */
+    boolean acceptsRelation(OSMTurnRelation relation);
```

Comment:
Maybe now that this one is only about turn restrictions, lets call it `acceptsTurnRelation` ?

Reply:
ok